src/web/modules/gateway_accounts/detail.njk

```diff
@@ -204,6 +204,11 @@
         href: "/gateway_accounts/" + gatewayAccountId + "/email_branding"
       })
       }}
+      {{ govukButton({
```

Comment:
I wonder if in the longer term it's worth moving this under the same category as the "Enable MOTO payments" section beneath this. Potentially making this less prominent in the context of that being enable/ disabled would help avoid confusion.

This seems to be a pattern for now so nothing this PR has to decide.

Reply:
Yeah, I think the navigation needs a bit of a rethink (you have to go to 3 different places to configure everything necessary for agent-initiated MOTO payments to work!) but I just followed the existing conventions for now.